types/media-manager/mediaManager.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  PipecatClientOptions,
+  RTVIEventCallbacks,
+  Tracks,
+} from '@pipecat-ai/client-js';
+import { MediaDeviceInfo } from '@daily-co/react-native-webrtc';
+
+export abstract class MediaManager {
```

Comment:
Is there anything other than `DailyMediaManager` that implements this? Is the idea that someday we may want to drop small WebRTC's dependency on Daily for media management?